.circleci/continue_config.yml

```diff
@@ -671,6 +676,29 @@ jobs:
                 paths:
                   - ~/.gradle
 
+  configurable-integration-test-spark:
+    working_directory: ~/openlineage/
+    machine:
+      image: ubuntu-2004:current
+    resource_class: large
+    steps:
+      - *checkout_project_root
+      - when:
+          condition:
+            equal: [ active, << pipeline.parameters.nightly-run >> ]
```

Comment:
Not sure why this needs to be true?

Reply:
There are some tests which I consider heavy enough so that they shouldn't be run with each commit. There are also tests, like databricks tests, which use external resources and should not be run each commit because of race condition possible when accessing external resources. Nightly builds can be a remedy to those problems. 

Currently, it is difficult to modify CI because there is a configuration flow which uses complex logic to create continuation workflow out of other workflow configuration files. CircleCI (up to my knowledge) does not accept any more workflows in main `config.yml` apart from configuration step + continuation workflow. 

Based on that, I decided to run the same workflow with extra param `nightly-run` = `active`. Then each each job, that is expected to run nightly and not on each commit, has to contain such running condition. 

To sum up: a nightly build is a regular build + bunch of jobs that normally don't get triggered bcz of such condition.

Reply:
to the last point, wouldn't it be better to have nightly build only run tests that are not run during regular builds? The whole point is that it would run on `main` code that has regular build already ran on it.